.github/workflows/deploy-happy-stack.yml

```diff
@@ -1,6 +1,9 @@
 name: Deploy Happy
 
-on: deployment
+on:
+  deployment:
+  repository_dispatch:
+    types: [deploy-images]
```

Comment:
maybe name this "deploy-happy-stack" for consistency, even if we don't like the name :)

Reply:
will do!